### PR TITLE
Blog post describing glosario

### PR DESCRIPTION
**Pull request by gvwilson**
Mostly pulled from the README of the glosario project.

cc @fmichonneau @zkamvar @ian-flores

### CODE REVIEW

File: _posts/2020/07/2020-07-27-announcing-glosario.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+in the page's YAML header
+in alphabetical order by term according to the rules for `glossary/language`.
+
+### Linking to definitions
+
+Amari adds an inline code block `` `r gdef('linear_model', 'modéle linéaire')` `` to her lesson.
+When she knits her document,
+the code block produces the HTML:
+
+```
```
Comment (zkamvar) [inline _posts/2020/07/2020-07-27-announcing-glosario.md]:
````suggestion
```html
````

Comment (gvwilson) [reply _posts/2020/07/2020-07-27-announcing-glosario.md]: :+1:

**Comment by fmichonneau** [comment]
@kmomar this is ready to publish. Thanks!

**Comment by serahkiburu** [comment]
thanks @fmichonneau - Omar is away from his desk, and I will be merging this shortly

**Pull request merged**